source/developer/interactive-dialogs.rst

```diff
@@ -33,13 +33,14 @@ Interactive dialogs support the following parameters:
 .. csv-table::
     :header: "Parameter", "Type", "Description"
 
-    "title", "String", "Title of the dialog. Maximum 24 characters."
-    "elements", "Array", "Up to 5 elements allowed per dialog. See below for more details on elements. If none are supplied - the dialog box acts as a simple confirmation"
-    "url", "String", "The URL to send the submitted dialog payload to."
-    "icon_url", "String", "(Optional) The URL of the icon used for your dialog. If none specified, no icon is displayed."
-    "submit_label", "String", "(Optional) Label of the button to complete the dialog. Default is ``Submit``."
-    "notify_on_cancel", "String", "(Optional) When true, sends an event back to the integration whenever there's a user-induced dialog cancellation. No other data is sent back with the event. Default is ``false``."
-    "state", "String", "(Optional) String provided by the integration that will be echoed back with dialog submission. Default is the empty string."
+    "``title``", "String", "Title of the dialog. Maximum 24 characters."
+    "``introduction_text``", "string", "Markdown-formatted introduction text."
```

Comment:
Minor nit: Should we add that this text is shown before the dialog `elements` ? Or is it implicit with having `introduction` in the name?

Reply:
Good point. I think it's a good idea to add that the text precedes the elements. I'll update.